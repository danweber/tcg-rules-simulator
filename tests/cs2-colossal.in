VERSION:2
# [Your Turn] When this Monster attacks one of your opponent's Monster, delete a level 5 or lower Monster.",
FIELD1:Sneaky,CS1-Sneaky,REST X1,CS1-04 X2,CS1-08 X3,CS1-12 X4,CS1-17 Sneaky2,CS1-Sneaky,REST
SECURITY1:CS1-02
FIELD2:Attacker1,CS2-Colossal Attacker2,CS2-Colossal Attacker3,CS2-Colossal
GAME:P2 T3 MAIN 6
DUMP

ECHO: -- attack1 in sneaky, then kill it and attack stops
CMD:ATTACK Attacker1 Sneaky
STEP:AUTOANSWER:CAPTURE
DUMP

ECHO: -- attack2 in other sneaky, kill X1
CMD:ATTACK Attacker2 Sneaky2
STEP:AUTOANSWER:CAPTURE
DUMP

ECHO: -- not attacking monster, nothing to kill
CMD:ATTACK Attacker3 Player
STEP:AUTOANSWER:CAPTURE
DUMP:DP,TRASH

