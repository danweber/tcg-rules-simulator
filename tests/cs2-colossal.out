FIELD1:Sneaky,CS1-06,REST X1,CS1-04 X2,CS1-08 X3,CS1-12 X4,CS1-17 Sneaky2,CS1-06,REST
FIELD2:Attacker1,CS2-12 Attacker2,CS2-12 Attacker3,CS2-12
GAME:P2 T3 MAIN 6
ECHO: -- attack1 in sneaky, then kill it and attack stops
CAPTURE:Target Sneaky Cat 1
CAPTURE:Target Stone Shield 2
CAPTURE:Target Phoenix Defender 3
CAPTURE:Target Sneaky Cat 6
FIELD1:X1,CS1-04 X2,CS1-08 X3,CS1-12 X4,CS1-17 Sneaky2,CS1-06,REST
FIELD2:Attacker1,CS2-12,REST Attacker2,CS2-12 Attacker3,CS2-12
GAME:P2 T3 MAIN 6
ECHO: -- attack2 in other sneaky, kill X1
CAPTURE:Target Stone Shield 2
CAPTURE:Target Phoenix Defender 3
CAPTURE:Target Sneaky Cat 6
FIELD1:X2,CS1-08 X3,CS1-12 X4,CS1-17
FIELD2:Attacker1,CS2-12,REST Attacker2,CS2-12,REST Attacker3,CS2-12
GAME:P2 T3 MAIN 6
ECHO: -- not attacking monster, nothing to kill
FIELD1:X2,CS1-08,1000DP X3,CS1-12,8000DP X4,CS1-17,NaNDP
FIELD2:Attacker1,CS2-12,REST,7500DP Attacker2,CS2-12,REST,7500DP Attacker3,CS2-12,REST,7500DP
TRASH1:CS1-06 CS1-04 CS1-06 CS1-02
TRASH2:
GAME:P2 T3 MAIN 6
